curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/66507c72ee0e194fd011ff39.md

```diff
@@ -0,0 +1,69 @@
+---
+id: 66507c72ee0e194fd011ff39
+title: Step 51
+challengeType: 1
+dashedName: step-51
+---
+
+# --description--
+
+With that quick review complete, you should remove your `addTwoNumbers` function, `sum` variable, and log statement.
+
+# --hints--
+
+You should not have a `addTwoNumbers` function in your code.
+
+```js
+assert.notMatch(code, /addTwoNumbers/);
+```
+
+You should not have a `sum` variable in your code.
+
+```js
+assert.notMatch(code, /sum/);
+```
+
+You should not have a third log statement in your code.
```

Comment:
```suggestion
You should not log the `sum` variable.
```

I think "third log statement" might get misinterpreted as removing the bottom log statement.

Reply:
How about "you should remove the log statement added in the previous step".

and test can be `assert.notMatch(code, /console.log(sum)/)`  but this will pass if the camper removes only the `sum` variable.

Reply:
I initially had that exact sentence. I just thought "You should not log the `sum` variable" was simpler and more in line with the other assert messages. But either one works for me.

Not sure if we need to change the test, although if we only count the amount of log statements, that doesn't guarantee the correct one was removed.